pkg/operation/botanist/etcdencryption.go

```diff
@@ -78,7 +78,7 @@ func (b *Botanist) ApplyEncryptionConfiguration(ctx context.Context) error {
 		conf   *apiserverconfigv1.EncryptionConfiguration
 	)
 	if b.Shoot.ETCDEncryption == nil {
-		return errors.New("Could not find etcd encryption configuration in ShootState")
+		return errors.New("could not find etcd encryption configuration in ShootState")
```

Comment:
Why is this part of this PR? It seems unrelated

Reply:
I was getting errors saying error string should not be capitalized.
Shall I keep it as it is?

Reply:
Where was this error coming from?
I think you can keep it, just got curious why you changed it.

Reply:
I didn't open the file at all. But somehow VSCode was showing it.